lookup_tables/greynoise/advanced/noise_advanced.yml

```diff
@@ -117,6 +117,9 @@ LogTypeMap:
         - "$.protoPayload.requestMetadata.callerIP"
         - "$.httpRequest.remoteIP"
         - "$.httpRequest.serverIP"
+    - LogType: GitHub.Audit
+      Selectors:
+        - 'actor_ip'
```

Comment:
I see other logs that reference `p_any_ip_addresses`, that seems like something we could or perhaps should do here in place of the individual field. What are your thoughts,  @rleighton ?

Reply:
It is always better to map the real field. Some sources don't have that ability because some ips are in JSON columns, so they use the `p_any` fields.